ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestsPage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import {
+    Flex,
+    FlexItem,
+    PageSection,
+    Tab,
+    TabTitleText,
+    Tabs,
+    Title,
+} from '@patternfly/react-core';
+
+type TabKey = 'PENDING_APPROVALS' | 'APPROVED_DEFERRALS' | 'APPROVED_FALSE_POSITIVES';
+
+function ExceptionRequestsPage() {
+    const [activeTabKey, setActiveTabKey] = useState<TabKey>('PENDING_APPROVALS');
```

Comment:
What do you think about using the `useURLStringUnion` hook instead, so that we can track the tab in the URL and allow linking to the different sections?